airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp

```diff
@@ -216,7 +216,11 @@
         if (currentSong) {
             updateWindowTitle(currentSong);
             <c:if test="${model.notify}">
-            showNotification(currentSong);
+            if ('mediaSession' in top.playQueue.navigator) {
```

Comment:
minor nit, why check the frame's navigator?
why not just the parent's frame's?

Reply:
MediaSession needs to be updated from the same frame/iframe as the `<audio>` element, and I figured having the same object used everywhere would reduce the risk of misunderstanding?

Reply:
so then why not just `navigator`?
```
if ('mediaSession' in navigator) {
```